### ISSUE TITLE
BTC on sterorids

### ISSUE THREAD

**Issue by adulau** [MISP/misp-modules#268]
~~~~
2019-01-24 07:35:31,098 - misp-modules - ERROR - Something went wrong:
Traceback (most recent call last):
  File "/home/circl/git/misp-modules/misp_modules/__init__.py", line 201, in post
    response = yield tornado.gen.with_timeout(timeout, self.run_request(jsonpayload))
  File "/home/circl/git/misp-modules/experimental/lib/python3.4/site-packages/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/home/circl/git/misp-modules/experimental/lib/python3.4/site-packages/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/circl/git/misp-modules/misp_modules/__init__.py", line 189, in run_request
    response = mhandlers[x['module']].handler(q=jsonpayload)
  File "modules/expansion/btc_steroids.py", line 161, in handler
    if tx['value'] != 0 and tx['addr'] == btc:
KeyError: 'addr'
ERROR:misp-modules:Something went wrong:
Traceback (most recent call last):
  File "/home/circl/git/misp-modules/misp_modules/__init__.py", line 201, in post
    response = yield tornado.gen.with_timeout(timeout, self.run_request(jsonpayload))
  File "/home/circl/git/misp-modules/experimental/lib/python3.4/site-packages/tornado/gen.py", line 1015, in run
    value = future.result()
  File "/home/circl/git/misp-modules/experimental/lib/python3.4/site-packages/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/circl/git/misp-modules/misp_modules/__init__.py", line 189, in run_request
    response = mhandlers[x['module']].handler(q=jsonpayload)
  File "modules/expansion/btc_steroids.py", line 161, in handler
    if tx['value'] != 0 and tx['addr'] == btc:
KeyError: 'addr'

~~~~

**Comment by rommelfs** [comment]
working on it

**Comment by rommelfs** [comment]
fixed in c52b95cdbe05744d3402550ff20f1ca4c8ba3ba9

**Comment by adulau** [comment]
@rommelfs Thank you.
